docs/guide/date-and-time-handling.md

```diff
@@ -1,19 +1,27 @@
 # Date and time handling
 
 The formats for the default date and time parsing functions can be set using configuration options:
-- [dateFormats](../api/interfaces/configparams.md#dateformats),
-- [timeFormats](../api/interfaces/configparams.md#timeformats),
-- [nullYear](../api/interfaces/configparams.md#nullyear).
+- [`dateFormats`](../api/interfaces/configparams.md#dateformats),
+- [`timeFormats`](../api/interfaces/configparams.md#timeformats),
+- [`nullYear`](../api/interfaces/configparams.md#nullyear).
 
+## Example
 
-## Example with Chinese
+The following date and time format configuration is used by default:
```

Comment:
As suggested by you in another comment, maybe we should make it explicit that the default configuration is "European":

```suggestion
By default, HyperFormula uses European date and time format configuration:
```